shark/parser.py

```diff
@@ -44,8 +44,7 @@ def dir_file(path):
 parser.add_argument(
     "--repro_dir",
     help="Directory to which module files will be saved for reproduction or debugging.",
-    type=dir_path,
-    default="./repro_dir/",
+    default="./tmp/",
```

Comment:
Can we then add `./shark_tmp` into `.gitignore`?

Reply:
How